travis-build.bash

```diff
@@ -20,18 +21,10 @@ function site-build() {
         return 1
     fi
     
-    local i
-    for (( i=0; i < 4; i++ )); do
-        # the logo img is added by material theme, so ignore it not having alt
-        if bundle exec htmlproofer ./site --alt-ignore '/.*\/atomist-logo-horiz-reversed.svg$/' \
-        --url-ignore "/api.github.com/,/jenkins.io/"
-        then
-            return 0
-        else
-            err "HTMLProofer attempt $i failed"
-        fi
-    done
-    return 1
+    if ! ./htmlproof.bash; then
```

Comment:
Why bother to move this into another file?

Reply:
so I can call it locally